src/Calculator/Views/MainPage.xaml

```diff
@@ -28,6 +28,7 @@
         <Style x:Key="AboutFlyoutPresenterStyle" TargetType="FlyoutPresenter">
             <Setter Property="IsTabStop" Value="False"/>
             <Setter Property="AutomationProperties.AccessibilityView" Value="Raw"/>
+            <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled"/>
```

Comment:
Why is this necessary? There is no ScrollViewer in the AboutFlyout.

Reply:
The default control template of FlyoutPresenter contains a ScrollViewer.

<img src="https://user-images.githubusercontent.com/10975439/101838570-de4b7480-3b0e-11eb-97a5-a72c2a5cef1d.png" alt="Control Template" width="600">

This is what it looks like if the ScrollViewer property setter is removed:
<img src="https://user-images.githubusercontent.com/10975439/101838760-32eeef80-3b0f-11eb-90b0-712376e1badc.png" alt="About Page with Scroll" width="400">

Reply:
Ah, ok. That makes sense. Good work!